data/txHandler_test.go

```diff
@@ -18,19 +18,18 @@ package data
 
 import (
 	"fmt"
-	"github.com/algorand/go-algorand/components/mocks"
 	"math/rand"
 	"testing"
 
-	"github.com/stretchr/testify/require"
-
+	"github.com/algorand/go-algorand/components/mocks"
 	"github.com/algorand/go-algorand/crypto"
 	"github.com/algorand/go-algorand/data/basics"
 	"github.com/algorand/go-algorand/data/pools"
 	"github.com/algorand/go-algorand/data/transactions"
 	"github.com/algorand/go-algorand/logging"
 	"github.com/algorand/go-algorand/protocol"
 	"github.com/algorand/go-algorand/util/execpool"
+	"github.com/stretchr/testify/require"
```

Comment:
this line should be moved back.. part of our coding conventions:
imports goes into three groups:
1. go imports
2. non-algorand imports
3. algorand imports

within each group, items are lexicographically ordered.